deployments.json

```diff
@@ -714,6 +709,16 @@
     "chainId": 314,
     "url": "https://filfox.info/en/address/0xcA11bde05977b3631167028862bE2a173976CA11"
   },
+  {
+    "name": "Astar zkEVM",
+    "chainId": 3776,
+    "url": "https://astar-zkevm.explorer.startale.com/address/0x36eabf148272BA81A5225C6a3637972F0EE17771?tab=contract"
```

Comment:
Since this one is at a different address, we'll need the `address` key like we have for zkSync Era